app/src/androidTest/java/chat/rocket/android/authentication/server/ui/ServerFragmentTest.kt

```diff
@@ -0,0 +1,52 @@
+package chat.rocket.android.authentication.server.ui
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.*
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.*
+import androidx.test.filters.LargeTest
+import androidx.test.rule.ActivityTestRule
+import chat.rocket.android.R
+import chat.rocket.android.analytics.event.ScreenViewEvent
+import chat.rocket.android.authentication.ui.AuthenticationActivity
+import chat.rocket.android.util.extensions.addFragmentBackStack
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+
+@LargeTest
+class ServerFragmentTest{
+
+    private val server = "open.rocket.chat"
```

Comment:
Do you think we should maybe have a config file to keep these kinds of infrastructure variables all in one place?

Reply:
Sure @ear-dev I will create one 👍